### PR TITLE
docs(core): :memo: corefilepicker has documentation has been updated

### LINKED ISSUE #331: docs: CoreFilePicker documentation need to be updated
## Description

- [ ] CoreFilePicker documentation needed to be updated according to new format

**Image**

image

### PR DESCRIPTION
**Pull request by arkaprota**
## Description

CoreFilePicker documentation has been updated to new format

Ref: #331

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Screenshots (if applicable)

image

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: app/components/component-docs/inputs/CoreFilePicker.docs.js [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+      component={CoreFilePicker}
+      description="The CoreFilePicker helps to directly upload file."
+      samples={
+        <>
+          <CodeSample
+            title={"Label & Helper Text"}
+            description="CoreFilePicker can use label and helper text props. Here is a example of it"
+            code={"<CoreFilePicker label=\"Profile Photo\" helperText=\"JPEG and PNG Formate Only\"></CoreFilePicker>"}
+            expandedCode={`import { CoreFilePicker } from "@wrappid/core";
+export default function BasicCoreBox() {
```
Comment (anantakumarghosh) [inline app/components/component-docs/inputs/CoreFilePicker.docs.js:18]: use suitable function name in expandedCode.

**Review by anantakumarghosh — CHANGES_REQUESTED** [review]
implement requested changes

**Review by techoneel — APPROVED** [review]
good to go

**Comment by anantakumarghosh** [comment]
> good to go

🤡

**Pull request merged**